### PR TITLE
Fix typo

### PR DESCRIPTION
**Pull request by carlossg**
@reviewbybees

### CODE REVIEW

**Comment by oleg-nenashev** [comment]
:bug:  I suspect it leads to the double reporting, because both classes have <code>@Extension</code> attribute

**Comment by carlossg** [comment]
Removed the `@Extension` from one of them. 

There are test errors but I'm getting those in master too

https://jenkins.ci.cloudbees.com/job/plugins/job/metrics-plugin/74/org.jenkins-ci.plugins$metrics/testReport/org.jvnet.hudson.test.junit/FailedTest/org_jvnet_hudson_test_JellyTestSuiteBuilder$JellyTestSuite/

File: src/main/java/jenkins/metrics/api/Metrics.java [hunk 1]
```diff
@@ -331,9 +331,18 @@ public void stop() throws Exception {
 
     /**
      * provides the health check related metrics.
+     * 
+     * @deprecated use HealthCheckMetricsProvider
+     */
+    @Deprecated
```
Comment (stephenc) [inline src/main/java/jenkins/metrics/api/Metrics.java]: `@Restricted(NoExternalUse.class)` to force people to remove the old reference when they update the dependency

**Comment by stephenc** [comment]
:bee:

**Comment by stephenc** [comment]
:+1: as maintainer

**Pull request merged**